niworkflows/interfaces/utils.py

```diff
@@ -956,7 +956,7 @@ def less_breakable(a_string):
     drop_columns = drop_columns or []
     additional_metadata = additional_metadata or {}
     try:
-        tsv_data = pd.read_csv(in_tsv, "\t")
+        tsv_data = pd.read_csv(in_tsv, delimiter="\t")
```

Comment:
It looks like delimiter is just an alias anyways
```suggestion
        tsv_data = pd.read_csv(in_tsv, sep="\t")
```